_posts/2019-10-29-sacred-geometry.md

```diff
@@ -0,0 +1,428 @@
+The 2 times tables.
+
+First, let's explain what I mean by a "clock". With the simpler example of the two times tables, let's say we map it on
+to a traditional analog clock face. So we have the numbers 1 -> 12 in arranged in a circle, and we take each one of
+those numbers and multiply it by 2, drawing a line from it that number, to what it equals when multiplied by 2. For the
+first half of our numbers, this is easy. We draw a line from 1 -> 2, from 2 -> 4, from 3 -> 6. But what happens when we
+hit 7? or anything higher than that? That is where the clock aspect comes in. If it is currently 2 hours past noon, what
+time is? 2 pm. So when we multiply 7 by 2, we go to 12, and then we go 2 more past that, so the point on the clock for 
+represents the number 2, but it also represents 14, and 26, and 38, and so on. This "clock" characteristic is an 
+embodiment of modulus division. So what we are actually doing is taking all the points on our clock, multiplying them by
+2, and modulo-dividing them by 12 to get the point at which they land on the clock. So for our 2 times tables on a 12 
+point clock, this is where our lines would go:
+```
+| Input | Output | Location |
+|-------|--------|----------|
+| 1     | 2      | 2        |
+| 2     | 4      | 4        |
+| 3     | 6      | 6        |
+| 4     | 8      | 8        |
+| 5     | 10     | 10       |
+| 6     | 12     | 12       |
+| 7     | 14     | 2        |
+| 8     | 16     | 4        |
+| 9     | 18     | 6        |
+| 10    | 20     | 8        |
+| 11    | 22     | 10       |
+| 12    | 24     | 12       |
+```
+You can see our example "clock face" below here, on the left. Next to it is the same 2 times tables, but with 48 lines. 
+Then we have the three times tables, and then the 4 times tables next to that. Interestingly, the "petals" of the 
+flower-like shapes in the images always have 1 less petal than their times table, i.e. 3 times tables has 2 petals, 4 
+times tables has three petals, and so on.
+
+<div style="text-align: center;">
+<div style="width: 175px; display: inline-block;">
+    <img src="/assets/images/articles/tt_two.svg">
+</div>
+<div style="width: 175px; display: inline-block;">
+    <img src="/assets/images/articles/tt_more_lines.svg">
+</div>
+<div style="width: 175px; display: inline-block;">
+    <img src="/assets/images/articles/tt_three.svg">
+</div>
+<div style="width: 175px; display: inline-block;">
+    <img src="/assets/images/articles/tt_four.svg">
+</div>
+</div>
+
+The next bit of math we will need is some small understanding of polar coordinates. Doing the math in polar coordinates 
+will actually handle the modulus logic for us, since it behaves just like our aforementioned clock anyway. If you take a
+polar coordinate and add 2&pi; to it (or Tau(&tau;), depending on 
+[which circle constant you prefer](https://www.youtube.com/watch?v=ZPv1UV0rD8U)) you end up right back where you 
```

Comment:
Not sure about @jimmykodes's or @ckcollab's preferences here, but this may direct the user off of the blog to YouTube. If add some code in the ending script, this link will open up in another tab. (See proposed code in last <script> section)

Reply:
good catch. turned these into `<a target="_blank">` links instead.